app/models/location.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class Location < ApplicationRecord
+  include Discard::Model
+  # TODO: rename disabled_at column to discarded_at and remove this line
```

Comment:
not sure we should rename the column - suppliers and the FE are expecting an attribute named "disabled_at" and they would need to update their systems if we renamed it to "discarded_at"

Reply:
The column could be renamed without changing the serializer, we do that a lot, it just would be nice to be more idiomatic with the discard gem. I'm not sure why but there are a few models that use the gem and a lot more that have "roll your own" soft delete functionality using a `disabled_at` column. I think it would be nice in future to standardise all of that to use the gem.

Reply:
👍